BPMRFFEApp/Db/bpmrffe.db

```diff
@@ -292,11 +291,19 @@ record(waveform, "$(P)$(R)RFFEVersion-Cmd") {
 
 record(bo, "$(P)$(R)RFFERst-Cmd") {
     field(DESC, "Reset the device")
-    field(DTYP, "stream")
     field(SCAN, "Passive")
-    field(VAL, "0")
-    field(ZNAM, "reset")
-    field(ONAM, "reset")
-    field(OUT, "@bpmrffe.proto Rst $(PORT)")
+    field(ZNAM, "off")
+    field(ONAM, "on")
+    field(HIGH, "1")
+    field(FLNK, "$(P)$(R)RFFERst-Snd")
 }
 
+record(calcout, "$(P)$(R)RFFERst-Snd") {
```

Comment:
I'd rather not create a suffix here. I'm okay with it not having any suffixes (we consider PVs without a suffix to be private by default)